bootstrap-gateway/deployment.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gateway-deployment
+  labels:
+    app: gateway
+    hail.is/sha: "{{ code.sha }}"
+spec:
+  selector:
+    matchLabels:
+      app: gateway
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: gateway
+        hail.is/sha: "{{ code.sha }}"
+    spec:
+      priorityClassName: infrastructure
+      nodeSelector:
+        preemptible: "true"
+      tolerations:
+       - key: preemptible
+         value: "true"
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
```

Comment:
I think we should use a consistent spacing for yaml dashes. build.yaml seems to do one space. You have two spaces here and a couple other spots and also zero spaces (under `values`).

Reply:
Agreed.  When I'm paying attention, I always try to use one.